app/main.py

```diff
@@ -1,3 +1,57 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
```

Comment:
Code Quality Issue: It's good to check that the input `km` is a positive integer. If it's not, then you should raise a `ValueError`.